### PR TITLE
feat: this PR adds blog for EMG Armband.

### PR DESCRIPTION
**Pull request by vovw**
Added new blog post on EMG armband project

This PR adds a new blog post detailing the development of an EMG (electromyography) armband system. The post covers:

- Project overview and goals
- Hardware components and signal processing
- Data collection and machine learning approach
- Weekly progress updates
- Future plans for web integration

The post includes diagrams and images to illustrate key concepts. It also acknowledges project mentees and sponsorship from Upside Down Labs.

### CODE REVIEW

File: _posts/2020-09-14-using-jekyll-with-github-pages-1.md
Comment (SuperChamp234) [inline _posts/2020-09-14-using-jekyll-with-github-pages-1.md:1]: What was changed in this file?

File: assets/posts/.DS_Store
Comment (SuperChamp234) [inline assets/posts/.DS_Store:1]: Get rid of this file please

File: assets/posts/emg-armband/.DS_Store
Comment (SuperChamp234) [inline assets/posts/emg-armband/.DS_Store:1]: Get rid of this file please

**Review by SuperChamp234 — CHANGES_REQUESTED** [review]

**Comment by SuperChamp234** [comment]
Maintainers, please make sure to squash and merge when this is ready.

**Comment by SuperChamp234** [comment]
> * grammer and formatting done.
> 
>     * added the images
> 
>     * added the sponsor logo
> 
>     * added the sponsor links
> 
>     * added information about whats next
> 
>     * added link to the github repo

Please do try to add a better description, even if short.

File: _posts/2020-09-14-using-jekyll-with-github-pages-1.md
Comment (vovw) [reply _posts/2020-09-14-using-jekyll-with-github-pages-1.md:1]: accidentally added a new line

**Comment by RapidRoger18** [comment]
Add the updates on weekly/biweekly basis, so 4 weeks of updates.

**Comment by vovw** [comment]
@RapidRoger18 @SuperChamp234 updated the description for the pr and added the weekly updates. Looks good?

**Comment by SuperChamp234** [comment]
> @RapidRoger18 @SuperChamp234 updated the description for the pr and added the weekly updates. Looks good?

Yeah, looks good! I won't review the blog contents, that's all up to you....

File: _posts/2020-09-14-using-jekyll-with-github-pages-1.md
Comment (SuperChamp234) [reply _posts/2020-09-14-using-jekyll-with-github-pages-1.md:1]: Why was this resolved?

Comment (vovw) [reply _posts/2020-09-14-using-jekyll-with-github-pages-1.md:1]: done

**Pull request merged**